libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -197,10 +226,27 @@ void SITL_State::_parse_command_line(int argc, char * const argv[])
             break;
         case 'I': {
             _instance = atoi(gopt.optarg);
-            _base_port  += _instance * 10;
-            _rcout_port += _instance * 10;
-            _rcin_port  += _instance * 10;
-            _fg_view_port += _instance * 10;
+            if ( _base_port == BASE_PORT ) {
+                _base_port += _instance * 10;
+            }
+            if ( _rcout_port == RCOUT_PORT ) {
```

Comment:
Spaces after/before parenthesis
Also, there is no way this condition is false. Maybe add that option too?